app/templates/core/contact.pug

```diff
@@ -8,11 +8,15 @@ block modal-body-content
     span(data-i18n="contact.welcome")
     span.spl(data-i18n="contact.forum_prefix")
     a(href="https://discourse.codecombat.com/", data-i18n="contact.forum_page")
-    span(data-i18n="contact.forum_suffix")
-    span.spl.spr(data-i18n="contact.faq_prefix")
-    a(data-i18n="contact.faq", href="https://codecombat.zendesk.com/hc/en-us")
-    span(data-i18n="contact.faq_suffix")
-    | .
+    if me.isHomeUser()
+      span(data-i18n="contact.forum_suffix")
+      span.spl.spr(data-i18n="contact.faq_prefix")
+      a(data-i18n="contact.faq", href="https://codecombat.zendesk.com/hc/en-us")
```

Comment:
we want to show zendesk for other users. the Forum part is for home users only

It will be for non-home users:
`You can check out our FAQs here for additional assistance`

Reply:
oh okay